src/app/settings/Troubleshooting.tsx

```diff
@@ -48,7 +49,7 @@ export function TroubleshootingSettings() {
           {t('Settings.ExportProfile')}
         </button>
 
-        {($DIM_FLAVOR === 'dev' || (window as any).enableMockProfile) && (
+        {$DIM_FLAVOR === 'dev' && (
```

Comment:
Out of curiousity, why this removal?

Reply:
Nothing sets `enableMockProfile` as far as I can tell. Must be a leftover from something else.

Reply:
I think the idea is that we can run `enableMockProfile = true` in the browser console and that allows us to upload a profile dump without spinning up the webpack dev server.

Reply:
Oh, I see. I'll document that.